spec/lib/efile/nj/nj1040_calculator_spec.rb

```diff
@@ -163,6 +163,49 @@ def over_65_birth_year
     end
   end
 
+  describe 'line 31 - medical expenses' do
+    let(:intake) { create(:state_file_nj_intake, medical_expenses: 123_456)}
+
+    before do
+      allow(instance).to receive(:calculate_line_29).and_return gross_income
+      allow(instance).to receive(:calculate_line_13).and_return 1234
+      instance.calculate
+    end
+
+    context 'when gross income is 0' do
+      let(:gross_income) { 0 }
+      it 'sets line 31 to the entire cost of medical expenses' do
+        expect(instance.lines[:NJ1040_LINE_31].value).to eq(123_456)
+      end
+
+      it 'includes medical expenses in line 38' do
+        expect(instance.lines[:NJ1040_LINE_38].value).to eq(124_690)
+      end
+    end
+
+    context 'when gross income is 10_000' do
```

Comment:
[pebble] can we reframe these test context names to describe the case it represents rather than the specific value? From reading this, I'm not sure why 10,000 and 100,000,000 have this behavior. Looking at the ticket, I assume it's when income * .02 is less than the medical expenses?

Can we:
a) rephrase the context strings to explain this, ie "when medical expenses is greater than 2% of income" or something like that
b) test the edge cases, like when medical expenses is exactly the same as 2% income, one dollar above, and one dollar below